unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -912,6 +912,18 @@ public void SetEngineDebugPanel()
             engineDebugPanel.SetActive(true);
         }
 
+        public void DumpScenesLoadInfo()
+        {
+            Debug.unityLogger.logEnabled = true;
```

Comment:
can we save `logEnabled` value in a temporal variable to avoid forcing `false` at the end of this method? 
because it might cause a conflict with calling `SetDebug()` that sets `logEnabled = true`

Reply:
good catch!

Reply:
done